pages/Javascript-API.md

```diff
@@ -44,6 +44,39 @@ Each object contains sub-objects which correlate to each element's subelements a
 
 If changes are made to the `config` object, you must call `scene.updateConfig()` for them to take effect.
 
+####`scene.config.textures`
```

Comment:
Well, the "JS API", for which we need a better definition and documentation, is basically:

- The `config` property of the scene, which (with maybe a couple small issues) shouldn't require additional documentation other than to explain that it's a deserialized / run-time object version of the scene file, in the same format. It's basically the scene file but can be dynamically modified.
- A few public methods that can be used to affect the scene, such as (but not limited to) `load()`, `rebuild()`, `updateConfig()`, `setDataSource()`, `getFeatureAtPixel()`, `screenshot()`, etc. The general approach has been to keep the number of actual public methods as small as possible, and to funnel all other changes through the `config` object, because modifying that is conceptually the same as modifying the scene file.

Overall, I'd like to be very careful about which `scene` properties we are documenting, as I don't consider the majority of them to be public, and it's not unusual for implementation to change them from version to version. This should probably be a separate discussion/PR to clarify.